static/app/views/performance/traces/fieldRenderers.tsx

```diff
@@ -39,19 +58,101 @@ export function ProjectRenderer({projectSlug, hideName}: ProjectRendererProps) {
   );
 }
 
+export const TraceBreakdownContainer = styled('div')`
+  position: relative;
+  display: flex;
+  min-width: 150px;
+  height: ${ROW_HEIGHT - 2 * ROW_PADDING}px;
+  background-color: ${p => p.theme.gray100};
+`;
+
+const RectangleTraceBreakdown = styled(RowRectangle)`
+  position: relative;
+  width: 100%;
+`;
+
+export function TraceBreakdownRenderer({trace}: {trace: TraceResult<Field>}) {
+  const theme = useTheme();
+
+  return (
+    <TraceBreakdownContainer data-test-id="relative-ops-breakdown">
+      {trace.breakdowns.map(breakdown => {
+        return (
+          <SpanBreakdownSliceRenderer
+            key={breakdown.start + (breakdown.project ?? t('missing'))}
+            sliceName={breakdown.project}
+            sliceStart={breakdown.start}
+            sliceEnd={breakdown.end}
+            trace={trace}
+            theme={theme}
+          />
+        );
+      })}
+    </TraceBreakdownContainer>
+  );
+}
+
+export function SpanBreakdownSliceRenderer({
+  trace,
+  theme,
+  sliceName,
+  sliceStart,
+  sliceEnd,
+}: {
+  sliceEnd: number;
+  sliceName: string | null;
+  sliceStart: number;
+  theme: Theme;
+  trace: TraceResult<Field>;
+}) {
+  const traceDuration = trace.end - trace.start;
+
+  const sliceDuration = sliceEnd - sliceStart;
+
+  if (sliceDuration <= 0) {
+    return null;
+  }
+  const sliceColor = sliceName ? pickBarColor(sliceName) : theme.gray100;
+  const slicePercent = toPercent(sliceDuration / traceDuration);
+  const relativeSliceStart = sliceStart - trace.start;
+  const sliceOffset = toPercent(relativeSliceStart / traceDuration);
+  return (
+    <div style={{width: slicePercent, left: sliceOffset, position: 'absolute'}}>
+      <Tooltip
+        title={
+          <div>
+            <div>{sliceName}</div>
+            <div>
+              <Duration seconds={sliceDuration / 1000} fixedDigits={2} abbreviation />
```

Comment:
Any reason to prefer `Duration` over `PerformanceDuration`?

Reply:
Oh I was following the span op breakdown, I didn't see that